### ISSUE TITLE
Model YLDP001 1SE color mostly working

### ISSUE THREAD

**Issue by steeveno** [home-assistant/home-assistant.github.io#17765]
### Feedback

I am using the model mentioned above. It basically works, it just gets unavailable approx every 3 months.

### URL

https://www.home-assistant.io/integrations/yeelight/

### Version

2021.5.1

**Comment by rytilahti** [comment]
Are you sure it is an issue in the integration and not in the device? What does it say in the logs when this happens? Without logs we cannot do really anything, not even deduce if the issue could be caused by homeassistant and not the device itself...

**Comment by steeveno** [comment]
Hi, unfortunately, the logs say absolutely nothing. At the device it just says "not available" and it is greyed out... I found some other posts related to this problem with Yeelight, and some proposed to add it via HomeKit, but it seems that mine do not support an integration via HomeKit. I now deactivated the sharing via LAN (or how it is called) in the Yeelight app, turned of the electricity, activated sharing again, and now they work. But after some time, it starts again... 

I also heard that activating the music mode might help, so I did it in configuration.yaml like this (indent is skipped here):
`    192.168.1.205:`
`          name: DekoMitteUnten`
`          use_music_mode: true`

But it does not seem to help either. Maybe because this model is not listed in the list of known working models?

Thank you,
steeveno
